mobilizon_bots/models/publication.py

```diff
@@ -1,14 +1,23 @@
+from enum import IntEnum
+
 from tortoise import fields
 from tortoise.models import Model
 
-from mobilizon_bots.event.event import PublicationStatus
+
+class PublicationStatus(IntEnum):
+    WAITING = 1
+    FAILED = 2
```

Comment:
a 'sto punto ho il dubbio su cosa significhi "failed". è failed solo se tutte le pubblicazioni sono fallite?

Reply:
imho yes a `MobilizonEvent` should be considered failed iif its publications are all failed, but it's definitely debatable

Reply:
commentare la definizione di failed e partial

failed: almeno uno è fallito
partial: mix di completo e waiting

Reply:
fixare failed per adattarsi alla definizione